wa/instruments/proc_stat/__init__.py

```diff
@@ -0,0 +1,94 @@
+#    Copyright 2020 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import time
+from datetime import datetime, timedelta
+
+import pandas as pd
+
+from wa import Instrument, Parameter, File, InstrumentError
+
+
+class ProcStatCollector(Instrument):
+
+    name = 'proc_stat'
+    description = '''
+    Collect CPU load information from /proc/stat.
+    '''
+
+    parameters = [
+        Parameter('period', int, default=5,
+                    constraint=lambda x: x > 0,
+                    description='''
+                    Time (in seconds) between collections.
+                    '''),
```

Comment:
Could you align these lines to keep pep8 happy? 
(https://travis-ci.org/ARM-software/workload-automation/jobs/647303406)

Reply:
Ah my bad. I had an old version of `flake8` on my machine that didn't pick this up until I upgraded it. Fixed now.